### ISSUE TITLE
Add eventContextExtractor to enable chaining of spans between directly invoked lambdas ?

### ISSUE THREAD

**Issue by technimad-splunk** [signalfx/splunk-otel-lambda#309]
Issue https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1363 describes the case where context propagation isn't working for lambdas directly invoking other lambda's.
Solution mentioned there is to add a eventContentExtractor to the AwsLambdaInstrumentation.
```
        eventContextExtractor: (event, context) => {
          const eventContext = propagation.extract(otelContext.active(), context.clientContext?.Custom);

          return eventContext;
        },
```
The [default extractor](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/d3c8bd9de4f29e217a0a299174e00abd0f792fec/plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts#L426) relied on in this implementation does something else compared to the one mentioned in the issue linked above.
Can we include the solution mentioned in the ticket above to support tracing for directly invoked lambda's?

Relevant line in the wrapper: https://github.com/signalfx/splunk-otel-lambda/blob/cd150c55d352baa51309f156ed56b3dff4bdf914/nodejs/src/wrapper.ts#L105

**Comment by technimad-splunk** [comment]
This is addressed in #313

**Comment by johnbley** [comment]
Yep, thanks for digging into that!
